pkg/versioninfo/versioninfo.go

```diff
@@ -31,6 +31,7 @@ type Status struct {
 	Version        string `json:"version"`
 	GitHash        string `json:"git_hash"`
 	StartTimestamp int64  `json:"start_timestamp"`
+	RegionLoaded   bool   `json:"region_loaded"`
```

Comment:
What if the operator/tiup uses this API and PD doesn't provide this field? Will it block the upgrade process?

Reply:
After discussion, we will use /ready api directly.